nautobot/core/templates/inc/nav_menu.html

```diff
@@ -837,6 +847,40 @@
                     </span>
                 </div>
             </form>
+            {% comment %}
+            The following if statement means the Query tab is only rendered when the GraphiQL
+            is rendered.
+            {% endcomment %}
+            {% if  saved_graphiql_queries and perms.extras.view_graphqlquery %}
+
+                <ul class="nav navbar-nav navbar-right">
+                    <li class="dropdown">
+                        <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">
+                            <span>Queries </span>
+                            <span class="caret"></span>
+                        </a>
+                        <ul class="dropdown-menu">
+                            {% for query in saved_graphiql_queries %}
+                                <li>
+                                    <a href="#" onclick="reload('{{ query.query|iriencode }}', '{{ query.variables|iriencode }}')">{{ query.name }}</a>
```

Comment:
I thought I asked this before but I can't find the comment, so I'll ask now. :-) What's the significance of using `iriencode` here (which isn't used anywhere else in Nautobot that I can see) instead of `urlencode` (which is used in a number of other places already)?